### ISSUE TITLE
eclipse m2e lifecycle-mapping configured wrong

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon-ide#150]
In my latest tests my eclipse m2e config is broken. The lifecycle-mapping config is set to:
```
/projects/devon//projects/devon/workspaces/main/devon-ide/settings/src/main/settings/eclipse/lifecycle-mapping-metadata.xml
```
See:
https://github.com/devonfw/devon-ide/blob/199ba7ab931e293fd8c8629ef2535b40622d36e7/settings/src/main/settings/eclipse/workspace/update/.metadata/.plugins/org.eclipse.core.runtime/.settings/org.eclipse.m2e.core.prefs#L7

IMHO already `SETTINGS_PATH` is already absolute now.

**Comment by hohwille** [comment]
Assumtion confirmed:
```
echo $SETTINGS_PATH
/projects/devon/workspaces/main/devon-ide/settings/src/main/settings
```

**Comment by hohwille** [comment]
Fixed with PR #148
